skel.ebuild

```diff
@@ -10,7 +10,9 @@
 # It is suggested that you use the latest EAPI approved by the Council.
 # The PMS contains specifications for all EAPIs. Eclasses will test for this
 # variable if they need to use features that are not universal in all EAPIs.
-EAPI=6
+# If an eclass doesn't support latest EAPI, use the previous EAPI instead. 
+EAPI=7
+
```

Comment:
Repoman will complain about two consecutive blank lines, so please avoid them in skel.ebuild too.

Reply:
But the skel.ebuild has more double blank lines, in those places where in new ebuild should be one. Specifically, between EAPI - DESCRIPTION, SRC_URI - LICENSE, IUSE - DEPEND. I thought it was by design. Should I remove the other double blank lines, too?

Reply:
Right, I've missed that. Leave the blank line in then.